addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2514,8 +2514,18 @@ registry.background = SnippetOptionWidget.extend({
             this.$target.css('background-image', `url('${widgetValue}')`);
             this.$target.addClass('oe_img_bg');
         } else {
-            this.$target.css('background-image', '');
-            this.$target.removeClass('oe_img_bg');
+            if (this.$target.is('.s_parallax_bg')) {
+                const $section = this.$target.closest('section');
+                $section.removeClass('parallax');
+                this.trigger_up('option_update', {
+                    optionNames: ['background', 'BackgroundPosition'],
+                    name: 'target',
+                    data: $section,
+                });
+            } else {
```

Comment:
I cannot see the word "parallax" elsewhere than in the parallax option. The parallax option handle the parallax feature. The background option handles the background feature. The color option handles the color feature.

As we discussed before, there are two options to handle this. The best one may be a bit tricky to implement so we choose the less good one: do the same way the background is removed when a color is chosen -> and that is not done by writing "remove background" in the color option. This is the background option listening the a custom event triggered by the color option and doing stuff when it catches it. Do the same in parallax, listening to a background custom event.